src/member.ts

```diff
@@ -107,4 +137,5 @@ export class Member implements IStateful<IMemberState> {
 export interface IMemberQueryOptions extends ICommonQueryOptions {
   address?: Address
   dao?: Address
+  [id: string]: any
```

Comment:
This is the pattern I use in other places, but I am not so happy with it, because it basically defeats the purpose of the type check on the options. If you have a better solution, I'd be glad to know :-)

Reply:
If the only reason it exists is to suppress the typescript compiler error on line #33, then there is another way. Adding `"suppressImplicitAnyIndexErrors": true` to your tsconfig.json will get rid of the error when you use the index operator on the object.

Reply:
that would be much better indeed, I opened an issue for it: https://github.com/daostack/client/issues/217